docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -1,9 +1,12 @@
 [[restart-cluster]]
-== Restart a cluster
+== Full-cluster restart and rolling restart
  
-There may be situations where you want to perform a full-cluster restart, which 
-means you shut down and restart all the nodes in the cluster. The following 
-description guides you through the process step by step.
+There may be {ref}/configuring-tls.html#tls-transport[situations where you want 
+to perform a full-cluster restart] or a rolling restart. In the first case, you 
+shut down and restart all the nodes in the cluster while in the second, you shut 
```

Comment:
I think stating "first" and "second" case here could be confusing. Let's explicitly call them full-cluster and rolling.